packages/react-table/src/components/Table/utils/decorators/selectable.tsx

```diff
@@ -43,7 +43,12 @@ export const selectable: ITransform = (
       : {
           checked: allRowsSelected,
           'aria-label': 'Select all rows'
-        })
+        }),
+    ...(rowData &&
+      rowData.disableCheckbox && {
+        disabled: true,
+        style: { cursor: 'not-allowed' }
```

Comment:
The code looks ok, but not sure about adding a style here. We may want to remove this and create a style in pf-core instead?

Reply:
:+1: